staging/src/k8s.io/apiserver/pkg/admission/metrics/metrics.go

```diff
@@ -149,11 +150,22 @@ func newAdmissionMetrics() *AdmissionMetrics {
 		},
 		[]string{"name", "type", "operation", "error_type", "rejection_code", "namespace"})
 
+	webhookRequest := metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Namespace:      namespace,
+			Subsystem:      subsystem,
+			Name:           "webhook_request_total",
+			Help:           "Admission webhook request total, identified by name and broken out for each admission type (validating or admit) and operation. Additional labels specify an HTTP status code. Codes greater than 600 are truncated to 600, to keep the metrics cardinality bounded.",
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{"name", "type", "operation", "code", "namespace"})
```

Comment:
On second thought, I think a rejection bool is needed here in order to distinguish between a rejected request with a certain code and a failed open request with the same code.
This could be done in a roundabout way by subtracting away rejections using the `rejection_count` metric, but that seems less than ideal.

Reply:
i'd add it now then, this is a candidate metric we'd want to promote to stable and if so we wouldn't want to make any breaking changes.

Reply:
Done.